### PR TITLE
Add bandit to banner tests

### PR DESCRIPTION
**Pull request by LAKSHMIRPILLAI**
## What does this change?

This PR is to add the the "Experiment Methodology " section to  banner tests
image

## How to test
Tested in CODE

## Before change
image

## After Change
image

### CODE REVIEW

File: public/src/components/channelManagement/epicTests/banditEditor.tsx [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-interface EpicBanditEditorProps {
-  test: EpicTest;
+interface BanditEditorProps {
+  test: EpicTest | BannerTest;
   isDisabled: boolean;
   onExperimentMethodologyChange: (isBanditTest?: boolean) => void;
 }
 
-export const BanditEditor: React.FC<EpicBanditEditorProps> = ({
+export const BanditEditor: React.FC<BanditEditorProps> = ({
```
Comment (tomrf1) [inline public/src/components/channelManagement/epicTests/banditEditor.tsx:12]: let's move this file out of `/epicTests`

File: public/src/models/banner.ts [hunk 2]
```diff
@@ -47,4 +47,5 @@ export interface BannerTest extends Test {
   deviceType?: DeviceType;
   campaignName?: string;
   contextTargeting: PageContextTargeting;
+  isBanditTest?: boolean;
```
Comment (tomrf1) [inline public/src/models/banner.ts:50]: how about adding this field to the `Test` model instead?

File: public/src/components/channelManagement/bannerTests/bannerTestEditor.tsx [hunk 3]
```diff
@@ -32,6 +32,9 @@ import {
 } from '../../../utils/requests';
 import TestEditorContextTargeting from '../testEditorContextTargeting';
 import { getDesignForVariant } from '../../../utils/bannerDesigns';
+import { BanditEditor } from '../banditEditor';
+
+const BANDIT_FEATURE_SWITCH = true;
```
Comment (tomrf1) [inline public/src/components/channelManagement/bannerTests/bannerTestEditor.tsx:37]: Do we need this?

Comment (LAKSHMIRPILLAI) [reply public/src/components/channelManagement/bannerTests/bannerTestEditor.tsx]: Removed that as we are going to keep bandit as a permanent feature

File: public/src/components/channelManagement/epicTests/testEditor.tsx [hunk 4]
```diff
@@ -34,11 +34,9 @@ import { useStyles } from '../helpers/testEditorStyles';
 import { EpicTestPreviewButton } from './testPreview';
 import { ValidatedTestEditor, ValidatedTestEditorProps } from '../validatedTestEditor';
 import { TestEditorProps } from '../testsForm';
-import { BanditEditor } from './banditEditor';
+import { BanditEditor } from '../banditEditor';
 import { AnalyticsButton } from '../AnalyticsButton';
 
-const BANDIT_FEATURE_SWITCH = true;
```
Comment (tomrf1) [inline public/src/components/channelManagement/epicTests/testEditor.tsx:40]: 👍 thanks!

**Review by tomrf1 — APPROVED** [review]
👍

**Pull request merged**